frontend/src/metabase/query_builder/components/expressions/Editor/custom-tooltip.tsx

```diff
@@ -0,0 +1,131 @@
+import {
+  type EditorState,
+  type Extension,
+  StateField,
```

Comment:
:x: Help tooltip causes body overflow when it does not fit the page.

![image](https://github.com/user-attachments/assets/df20da8e-e3ba-4481-83bb-a65db0baff38)

Same with suggestions popover is not scrollable, it causes body to overflow.
https://github.com/user-attachments/assets/77627012-dcd8-48d0-b87c-17e878bc87c9